config/dbstructure.config.php

```diff
@@ -34,7 +34,7 @@
 use Friendica\Database\DBA;
 
 if (!defined('DB_UPDATE_VERSION')) {
-	define('DB_UPDATE_VERSION', 1307);
+	define('DB_UPDATE_VERSION', 1308);
```

Comment:
This should be bumped to `1309`, I'm surprised you didn't get a conflict after #6939 was merged.